src/tests/data/tls-policy/strict.txt

```diff
@@ -1,22 +1,28 @@
-allow_tls10 = false
-allow_tls11 = false
 allow_tls12 = true
 allow_tls13 = false
-allow_dtls10 = false
 allow_dtls12 = true
+allow_ssl_key_log_file = false
 ciphers = ChaCha20Poly1305 AES-256/GCM AES-128/GCM
 macs = AEAD
 signature_hashes = SHA-512 SHA-384
 signature_methods = ECDSA RSA
 key_exchange_methods = ECDH
-key_exchange_groups = x25519 x448 secp256r1 brainpool256r1 secp384r1 brainpool384r1 secp521r1 brainpool512r1 ffdhe/ietf/2048 ffdhe/ietf/3072 ffdhe/ietf/4096 ffdhe/ietf/6144 ffdhe/ietf/8192
+key_exchange_groups = x25519/Kyber-768-r3 x25519 secp256r1 x448 secp384r1 secp521r1 brainpool256r1 brainpool384r1 brainpool512r1 ffdhe/ietf/2048 ffdhe/ietf/3072
```

Comment:
```suggestion
key_exchange_groups = x25519/ML-KEM-768 x25519 secp256r1 x448 secp384r1 secp521r1 brainpool256r1 brainpool384r1 brainpool512r1 ffdhe/ietf/2048 ffdhe/ietf/3072
```

Reply:
We're not really offering Kyber here. It being in the policy text is a leftover from earlier iterations of the PR. But now the Kyber/MKLEM suites are being filtered out and I don't understand why.

Reply:
Something was behaving weirdly here because the tests for me passed with Kyber in the expected policy text (even though the policy doesn't include Kyber anymore) and then started failing when I switched it to ML-KEM. No idea what happened, went away after a clean rebuild.